yarn.lock

```diff
@@ -5398,6 +5401,13 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@marijn/find-cluster-break@npm:^1.0.0":
+  version: 1.0.1
+  resolution: "@marijn/find-cluster-break@npm:1.0.1"
```

Comment:
I think bumping resolution to 1.0.2 should help as per https://github.com/marijnh/find-cluster-break/issues/1

Reply:
nice, thanks, let's try that.